tools/python/paladin_claiming/recycle_bribes.py

```diff
@@ -0,0 +1,195 @@
+import json
+from dotenv import load_dotenv
+from typing import List
+import sys
+from pathlib import Path
+import os
+from datetime import datetime
+
+sys.path.append(str(Path(__file__).parent.parent))
+
+import requests
+from web3 import Web3
+from helpers.hh_bribs import get_hh_aura_target
+from bal_addresses import AddrBook
+from bal_tools.safe_tx_builder import SafeTxBuilder, SafeContract
+from bal_tools import Web3RpcByChain
+
+
+load_dotenv()
+
+SCRIPT_DIR = Path(__file__).parent
+CLAIM_OUTPUT_DIR = SCRIPT_DIR / "claim_output"
+PAYLOAD_DIR = SCRIPT_DIR / "payload"
+
+CLAIM_OUTPUT_DIR.mkdir(parents=True, exist_ok=True)
+PAYLOAD_DIR.mkdir(parents=True, exist_ok=True)
+
+CURRENT_DATE = datetime.utcnow().strftime("%Y-%m-%d")
+
+w3_by_chain = Web3RpcByChain(os.getenv("DRPC_KEY"))
+flatbook_mainnet = AddrBook("mainnet").flatbook
+flatbook_arb = AddrBook("arbitrum").flatbook
+
+omni_safe = flatbook_mainnet["multisigs/vote_incentive_recycling"]
+
+CHAIN_ADDRS = {
+    "mainnet": {
+        "paladin_distributor": "0x1F7b4Bf0CD21c1FBC4F1d995BA0608fDfC992aF4",
```

Comment:
id suggest getting all hardcoded addys in this script to address book (also the ones for arb1)

Reply:
can track here: https://github.com/BalancerMaxis/bal_addresses/pull/513

Reply:
ok great, i left a comment. once merged we can get rid of the hardcoded addys here

Reply:
f49a48b